config/locales/de.yml

```diff
@@ -43,7 +54,7 @@ de:
         name: 'Name'
         link: 'https://'
       labels:
-        image: 'Bild (Datei-Typ: PNG or SVG, Empfohlene Größe: 1140x100)'
+        image: 'Bild / Thumbnail (Datei-Typ: PNG or SVG, empfohlene Größe: 1000x667)'
```

Comment:
Noch eine Kleinigkeit (nice to have): Magst du dann gleich noch mal ein gutes Beispielbild in lib/generators/cross_promotion_app/templates/images legen? 

Bisher ist dort das b2b_banner drin.